event/serializers.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework import serializers
+from . import models
+
+
+class EventImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.EventImage
+        fields = "__all__"
+
+
+class EventLocationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.EventLocation
+        fields = "__all__"
+
+
+class HotTopicSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.HotTopic
+        fields = "__all__"
+
+
+class SpeakerSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.Speaker
+        fields = "__all__"
+
+
+class ScheduleSerializer(serializers.ModelSerializer):
+    speakers = SpeakerSerializer(many=True)
+
+    class Meta:
+        model = models.Schedule
+        fields = "__all__"
+
+
+class EventSerializer(serializers.ModelSerializer):
```

Comment:
Ensure creation and updating of Event model 

eg:
```python
def update(self, instance, validated_data):
        instance.title = validated_data.get('title', instance.title)
        instance.start_date = validated_data.get('start_date', instance.start_date)
        instance.end_date = validated_data.get('end_date', instance.end_date)
        # similarly aru fields
        instance.save()
        return instance
```

Reply:
No my theory is that we will only create events from Admin Panel